### ISSUE TITLE
Install Tensorflow C++ API

### ISSUE THREAD

**Issue by AbduElrahmanRezk** [tensorflow/tensorflow#35870]
**System information**
- OS Platform and Distribution :  Linux Ubuntu 18.04
- TensorFlow installed from : Anaconda
- TensorFlow version: 1.15
- Python version:3.7(anaconda)
- Installed using virtualenv? pip? conda?:conda
- Bazel version (if compiling from source):1.2.1
- GCC/Compiler version (if compiling from source):7.4.0
- CUDA/cuDNN version: cuda10.1 cudnn 7.6
- GPU model and memory:  NVIDIA-SMI 435.21       Driver Version: 435.21 

**Describe the problem**
I want to install tensorflow c api and start training my models in c as in python 
i followed the instruction mentioned here [https://www.tensorflow.org/install/source](url)
but after the making configuration by bazel i cannot get my code running 
my code 
`#include "tensorflow/cc/ops/const_op.h"
#include "tensorflow/cc/ops/image_ops.h"
#include "tensorflow/cc/ops/standard_ops.h"
#include "tensorflow/core/framework/graph.pb.h"
#include "tensorflow/core/framework/tensor.h"
#include "tensorflow/cc/framework/gradients.h"
#include "tensorflow/core/graph/default_device.h"
#include "tensorflow/core/graph/graph_def_builder.h"
#include "tensorflow/core/lib/core/errors.h"
#include "tensorflow/core/lib/core/stringpiece.h"
#include "tensorflow/core/lib/core/threadpool.h"
#include "tensorflow/core/lib/io/path.h"
#include "tensorflow/core/lib/strings/stringprintf.h"
#include "tensorflow/core/platform/init_main.h"
#include "tensorflow/core/platform/logging.h"
#include "tensorflow/core/platform/types.h"
#include "tensorflow/core/public/session.h"
#include "tensorflow/core/util/command_line_flags.h"
#include "tensorflow/core/platform/load_library.h"`
it gives this dependency errors due to missing files (.h files)
I have two questions
1- where should be the header files placed ?
2- how can i get the missing files which is not here [https://github.com/tensorflow/tensorflow/tree/master/tensorflow](url) ?

**Comment by amahendrakar** [comment]
Any updates regarding this issue? Thanks!

**Comment by amahendrakar** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
